iocore/net/SSLNetVConnection.cc

```diff
@@ -51,11 +52,11 @@ using namespace std::literals;
 #include <openssl/async.h>
 #endif
 
-#if !TS_USE_SET_RBIO
-// Defined in SSLInternal.c, should probably make a separate include
-// file for this at some point
-void SSL_set0_rbio(SSL *ssl, BIO *rbio);
-#endif
+// #if !TS_USE_SET_RBIO
+// // Defined in SSLInternal.c, should probably make a separate include
+// // file for this at some point
+// void SSL_set0_rbio(SSL *ssl, BIO *rbio);
+// #endif
```

Comment:
Go ahead and delete this since you moved it into the SSLInternal.h header as it should be.